internal/util/util.go

```diff
@@ -219,7 +219,61 @@ func FSize(f *os.File) int64 {
 // IsWordChar returns whether or not the string is a 'word character'
 // Word characters are defined as numbers, letters, or '_'
 func IsWordChar(r rune) bool {
-	return unicode.IsLetter(r) || unicode.IsNumber(r) || r == '_'
+	return IsAlphanumeric(r) || IsWordDelimiter(r)
+}
+
+// IsUpperWordChar returns whether or not the string is a 'upper word character'
+// Upper word characters are defined as numbers, upper-case letters, or delimiters ([A-Z0-9_])
```

Comment:
There references to regex patterns are not quite accurate, since `unicode.IsLetter()` returns true not only for Latin (ascii) letters.

Reply:
True.
For `IsUpperWordChar` I could change it from `([A-Z0-9_])` to: `[\W]` and 
for `IsLowerWordChar` I could change it from `([a-z0-9_])` to: `[\w]`

Or get rid of all regex hints at all.

Reply:
I'd vote for getting rid of them.

Reply:
Agreed.